src/UglyToad.PdfPig.Tokenization/NumericTokenizer.cs

```diff
@@ -153,7 +153,7 @@ public bool TryTokenize(byte currentByte, IInputBytes inputBytes, out IToken tok
                         token = NumericToken.OneThousand;
                         return true;
                     default:
-                        if (!decimal.TryParse(str, NumberStyles.Any, CultureInfo.InvariantCulture, out var value))
+                        if (!double.TryParse(str, NumberStyles.Any, CultureInfo.InvariantCulture, out var value))
```

Comment:
Random thought while skimming this, is it possible `double` supports/returns `NaN` where previously decimal wouldn't? I think the integration tests would catch that tbh so it's probably fine.

Reply:
I think you're right. I'm pretty sure that is `str` = "NaN", then `double.TryParse()` would return `NaN`. But as you said, I think we are good for the moment as the integration tests would have caught any issue.

Good to keep in mind that in can happen though